docs/setup/new_environment.md

```diff
@@ -207,7 +222,7 @@ Even though we will be running all commands locally, we still need to add the us
 1. Copy the example fab config file:
 
     ``` bash
-    $ cp ~/commcare-cloud/src/commcare_cloud/fab/config.example.py ~/commcare-cloud/src/commcare_cloud/fab/config.py
+    $ cp ~/commcare-cloud/src/commcare_cloud/config.example.py ~/commcare-cloud/src/commcare_cloud/fab/config.py
```

Comment:
It's weird that the example file wouldn't be in the same directory that it should end up in. I think instead of this, moving the example into `commcare_cloud/fab/` might be better.

Reply:
Ok, I'll just move it back in then. I thought it was done so on purpose, but based on Ethan's comment it looks like an oops :)

Reply:
I'm confused about which one we're talking about, but the correct location for the file (and as Farid said, the example file) is in the root dir:
https://github.com/dimagi/commcare-cloud/blob/1aba598c3878d539eae70e3b51ebfe8f9a5a146f/src/commcare_cloud/github.py#L37-L42

Reply:
My understanding is that the destination file should also be in the root dir, ie:
```suggestion
    $ cp ~/commcare-cloud/src/commcare_cloud/config.example.py ~/commcare-cloud/src/commcare_cloud/config.py
```
Was this not your experience when going through the setup?

Reply:
O, right! Sorry I missed you completely for a moment. Mmm...yes that would make 100% sense.